plugins/nf-co2footprint/src/main/nextflow/co2footprint/CO2FootprintFactory.groovy

```diff
@@ -183,6 +183,7 @@ class CO2FootprintFactory implements TraceObserverFactory {
          */
         // nm: size of memory available [GB] -> requested memory
         Long memory = trace.get('memory') as Long
+        memory = 1000000
```

Comment:
@mirpedrol where is this coming from? was this for testing maybe?

Reply:
oh sorry, it was indeed for testing! I committed it by mistake

Reply:
but the rest can stay at it is? Then I will use it